### PR TITLE
feat: add tests

### PR DESCRIPTION
**Pull request by clauBv23**

### CODE REVIEW

File: packages/contracts/test/10_unit-testing/12_plugin-setup.ts [hunk 1]
```diff
@@ -829,17 +848,42 @@ describe('TokenVotingSetup', function () {
...
           plugin,
           dao.address,
           AddressZero,
-          SET_TARGET_CONFIG_PERMISSION_ID
+          SET_TARGET_CONFIG_PERMISSION_ID,
         ],
       ]);
     });
+
+    it('returns the permissions expected for the update from build 3', async () => {
```
Comment (novaknole) [inline packages/contracts/test/10_unit-testing/12_plugin-setup.ts]: `From build` or `to build` ? from build suggests that it's upgrading from v3 and there's no v3 for multisig.

File: packages/contracts/test/30_regression-testing/31_upgradeability.ts [hunk 2]
```diff
@@ -117,6 +121,142 @@ describe('Upgrades', () => {
...
+    expect(await newTokenVoting.minApproval()).to.equal(defaultMinApproval);
+    expect((await newTokenVoting.getTargetConfig()).target).to.deep.equal(
+      defaultTargetConfig.target
+    );
+    expect((await newTokenVoting.getTargetConfig()).operation).to.deep.equal(
+      defaultTargetConfig.operation
+    );
+  });
+
+  it('from v1.2 with `initializeFrom`', async () => {
```
Comment (novaknole) [inline packages/contracts/test/30_regression-testing/31_upgradeability.ts]: same description as above:

```
upgrades from v1.2 with `initializeFrom`
```

File: packages/contracts/test/10_unit-testing/12_plugin-setup.ts (hunk 1 above)
Comment (clauBv23) [reply packages/contracts/test/10_unit-testing/12_plugin-setup.ts]: it is testing updating from build 3 to build 3, this is not realistic but it was to check that when upgrading from bigger versions, permissions list is empty

**Comment by novaknole** [comment]
Closed in favor of https://github.com/aragon/token-voting-plugin/pull/35

**Pull request closed without merging**